.travis.yml

```diff
@@ -29,15 +29,15 @@ matrix:
   fast_finish: true
   include:
     - compiler: ": ARM"
-      env: HOST=arm-linux-gnueabihf PACKAGES="g++-arm-linux-gnueabihf" DEP_OPTS="NO_QT=1" GOAL="install" BITCOIN_CONFIG="--enable-glibc-back-compat --enable-reduce-exports"
```

Comment:
.travis.yml isn't actually used, as shown by this bitrot. I can split it into another commit if necessary.